modules/release-notes/pages/relnotes.adoc

```diff
@@ -1,10 +1,306 @@
 = Release Notes for Couchbase Server 7.0
-:description: Couchbase Server 7.0 introduces scopes and collections that provide new database organizaion structures within Couchbase.
+:description: Couchbase Server 7.0 introduces scopes and collections, which provide new database organization structures within Couchbase.
 
 {description} It also brings new capabilities to the Query Service in addition to support for scopes and collections.
 
 Take a look at xref:introduction:whats-new.adoc[What's New?] for a list of new features and improvements in this release.
 
+[#release-702]
+== Release 7.0.2 (October 2021)
+
+Couchbase Server 7.0.2 was released in October 2021.
+This maintenance release contains new features and fixes to issues.
+
+[#new-features-improvements-702]
+=== New Features
+
+This section highlights the notable new features and improvements in this release.
+
+* TLS can be specified as mandatory for all internal and external cluster-communications &#8212; see xref:manage:manage-security/manage-tls.adoc[Manage On-the-Wire Security].
+
```

Comment:
We added a new feature to allow administrators to enable HSTS (HTTP Strict Transport Security)

Reply:
I have added "* HSTS (HTTP Strict Transport Security) can now be enabled."